### PR TITLE
Личный проект: доверяй, но проверяй

### PR DESCRIPTION
**Pull request by akzhar**
---
:mortar_board: [Личный проект: доверяй, но проверяй](https://up.htmlacademy.ru/javascript/17/user/490939/tasks/8)

:boom: https://htmlacademy-javascript.github.io/490939-keksobooking-17/4/

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -89,14 +108,35 @@ function fillPinCenterInAddress(pinObj) {
...
+  mapPinMain.removeEventListener('click', onMapPinMainClick);
+}
+
+function onTypeSelectChanged() {
+  var selectedOptionIndex = typeSelect.selectedIndex;
+  var selectedOption = typeSelect.querySelectorAll('option')[selectedOptionIndex];
+  var i = 0;
+  while (APARTMENT_TYPES[i].type !== selectedOption.value) {
+    i++;
+  }
```
Comment (rkhaslarov) [inline js/main.js]: weird approach, use filter method of the array )

Comment (rkhaslarov) [reply js/main.js]: or even findIndex =)

Comment (akzhar) [reply js/main.js]: fixed

File: js/main.js [hunk 2]
```diff
@@ -3,16 +3,33 @@
...
+  },
+  {
+    type: 'house',
+    minPrice: 5000
+  },
+  {
+    type: 'bungalo',
+    minPrice: 0
+  },
+];
```
Comment (rkhaslarov) [inline js/main.js]: why you don't put these values into the object? it gonna be way faster

Comment (akzhar) [reply js/main.js]: Is access to the object faster than acess to the array in general?

**Pull request merged**